### PR TITLE
 replacement of es5 functions to es6 class def feat P5.AudioIn and audiocontext

### PR DESCRIPTION
**Pull request by endurance21**
## area covered 
* src/audioIn
* src/audiocontext
* src/app

### CODE REVIEW

**Comment by endurance21** [comment]
@therewasaguy @kyle1james  please review this pr!

File: src/audioin.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ p5sound.inputSources = [];
  *
  *   function setup(){
  *    let cnv = createCanvas(100, 100);
- *    cnv.mousePressed(userStartAudio);
+ *    cnv.mousePressed(useraudiocontextStartAudio);
```
Comment (therewasaguy) [inline src/audioin.js:34]: not sure why this changed (or if you meant to change it)?

Comment (endurance21) [reply src/audioin.js:34]: oh! i dont need to make that happen! 
no worries , i ll be correcting it !

**Comment by endurance21** [comment]
@therewasaguy @kyle1james  ready to go ! ?

**Pull request merged**